projects/compose/koin-compose/src/wasmJsMain/kotlin/org/koin/compose/KoinApplication.wasmJS.kt

```diff
@@ -0,0 +1,16 @@
+package org.koin.compose
+
+import androidx.compose.runtime.Composable
+import org.koin.core.KoinApplication
+import org.koin.dsl.KoinConfiguration
+import org.koin.core.logger.Level
+import org.koin.dsl.koinConfiguration
+import org.koin.dsl.includes
+
+@Composable
+internal actual fun composeConfiguration(loggerLevel : Level,config : KoinApplication.() -> Unit) : KoinConfiguration {
+    return koinConfiguration {
```

Comment:
this builds configuration on every recomposition and will lead to performance issues